contracts/interfaces/IIndividualClaims.sol

```diff
@@ -122,7 +122,7 @@ interface IIndividualClaims {
 
   /* ========== EVENTS ========== */
 
-  event ClaimSubmitted(address user, uint claimId, uint coverId, uint productId);
+  event ClaimSubmitted(address indexed user, uint indexed claimId, uint coverId, uint productId);
```

Comment:
🔨  It should be `coverId` indexed instead of `claimId` as we'd need to filter the claims by cover.